projects/netgrif-components-core/src/lib/task-content/task-content/abstract-task-content.component.ts

```diff
@@ -450,7 +450,7 @@ export abstract class AbstractTaskContentComponent implements OnDestroy {
      */
     protected computeGridLayout(dataGroup: DataGroup, subgrid: Subgrid) {
         const localGrid: Array<Array<string>> = [];
-
+        dataGroup.fields = dataGroup.fields.sort((f1 , f2) => f1.layout.y - f2.layout.y)
```

Comment:
fields that are on the same row can still render in incorrect order, if the rendering batch does not include the entire row.

if `f1.y === f2.y`, then `f1.x - f2.x` should be returned.

such sort might already be implemented somewhere and the function could possibly be reused

Reply:
Corrected according to request, hence I did not found such function, so I added own implementation.